requirements.txt

```diff
@@ -2,9 +2,11 @@
 django==1.9
 djangorestframework==3.6.4
 django-cors-headers
-django-oauth-toolkit==0.11.0
-django-model-utils==3.0.0
 django-filter==1.0.4
+django-model-utils==3.0.0
+django-oauth-toolkit==0.11.0
```

Comment:
Why do you need this?

Reply:
This was previously added, I just alphabetised as you asked in earlier comment.

Reply:
I understand, but my original question remains: why does this service need an OAuth provider? We already have one on LMS.

Reply:
hmm, @yro can tell better I guess

Reply:
@clintonb There are "dumb" encode worker nodes that call this thing via web API, it's wholly independent from the LMS.

Reply:
@clintonb To be clear I'm not defending having oauth-toolkit as the "right" solution, just as the "currently in production" solution. I think this may create a little tech debt, but I've got a short term plan to use a different solution for this which will eliminate it.